resources/public/api/conf/1.0/employment_expenses_create_and_amend.yaml

```diff
@@ -6,33 +6,14 @@ description: |
   Amending an expense will override the expense and remove the "dateIgnored" flag from HMRC-provided expenses.
 
   ### Test data
-  <p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
-  <table>
-      <thead>
-          <tr>
-              <th>Header Value (Gov-Test-Scenario)</th>
-              <th>Scenario</th>
-          </tr>
-      </thead>
-      <tbody>
-          <tr>
-              <td><p>N/A - DEFAULT</p></td>
-              <td><p>Simulate a success response.</p></td>
-          </tr>
-          <tr>
-              <td><p>TAX_YEAR_NOT_ENDED</p></td>
-              <td><p>Simulates the scenario where the submission is for a tax year that has not ended.</p></td>
-          </tr>
-          <tr>
-              <td><p>NOT_FOUND</p></td>
-              <td><p>Scenario to simulate not found error.</p></td>
-          </tr>
-          <tr>
-              <td><p>STATEFUL</p></td>
-              <td><p>Performs a stateful create or update.</p></td>
-          </tr>
-      </tbody>
-  </table>
+    Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.
```

Comment:
Not related to the markdown change, but while you're there, could you fix the grammar in each of these intro sentences?  ("are only" rather than "is only")